packages/terra-graphs-docs/src/terra-dev-site/doc/Graphs/Combination.g/About.a.doc.mdx

```diff
@@ -0,0 +1,150 @@
+import CombinationGraph from '../../example/Graphs-Carbon/CombinationGraph/BasicCombinationGraphs/BarGraphWithLine?dev-site-example';
+import GraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault.js?dev-site-codeblock';
+import DataObject1 from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/barDataset.js?dev-site-codeblock';
+import DataObject2 from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineDataset1.js?dev-site-codeblock';
+require('details-polyfill')
+
+# Combination
+
+A combination graph is built by combining two different native graphs(line, bar, paired result etc) build using D3 based on standard design patterns.
+
+-   We can draw different combinations of graphs. i.e. We can draw a _Combination graph_ with _2 Bar graphs_ or _1 Bar graph_ and _1 line graph_ without ever needing to building those natively in Carbon.
+-   We can draw different combinations of graph in any order necessary.
+
+-   [Combination](#combination)
+    -   [Getting Started](#getting-started)
+    -   [Usage](#usage)
+    -   [Configuration Structure](#configuration-structure)
+    -   [Configuration properties](#configuration-properties)
+    -   [Dataset Structure](#dataset-structure)
+    -   [Dataset Properties](#dataset-properties)
+        -  [Constraints](#constraints)
+    -  [Example](#example)
+
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm i @cerner/carbon-graphs --save-dev`
+
+## Usage
+
+An instance of Combination graph is obtained as follows:
+
+```js
+Carbon.api.line(/* dataset1 */)
+Carbon.api.bar(/* dataset2 */)
+```
+A Combination graph can be plotted as follows:
+
+```js
+import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
+import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
+const graph = Carbon.api.graph(/* graph config JSON */);
+graph.loadContent(Carbon.api.bar(/* dataset 1 */));
+graph.loadContent(Carbon.api.line(/* dataset 2 */));
+
+// Multiple graph types can be added to a combination graph:
+graph.loadContent(Carbon.api.bar(/* dataset 1 */));
+graph.loadContent(Carbon.api.line(/* dataset 2 */));
+graph.loadContent(Carbon.api.pairedResult(/* dataset 2 */));
```

Comment:
this looks like duplicate information. Maybe update to the following:

```suggestion
import Carbon from '@cerner/carbon-graphs/dist/js/carbon-graphs.js';
import '@cerner/carbon-graphs/dist/css/carbon-graphs.css';
const graph = Carbon.api.graph(/* graph config JSON */);
graph.loadContent(Carbon.api.line(/* dataset 1 */));

// Additional graph types can be added:
graph.loadContent(Carbon.api.bar(/* dataset 2 */));
graph.loadContent(Carbon.api.pairedResult(/* dataset 3 */));
```

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/258/commits/26aa906c7e38cb1eb7634641d837ae7e46fef438).